docs/products/opensearch/dashboards/index.rst

```diff
@@ -0,0 +1,31 @@
+OpenSearch Dashboards
+======================
+
+OpenSearch Dashboards is both a visualisation tool for data inside an OpenSearch cluster and a user interface for OpenSearch plugins.
```

Comment:
would "for data in OpenSearch" still work? Simplifying a bit the wording

Reply:
I replaced with "for data in the cluster", a bit shorter and still precise.